pkgs/development/python-modules/beancount-plugin-utils/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, python3
+, fetchFromGitHub
+, buildPythonPackage
+}:
+
+buildPythonPackage rec {
+  pname = "beancount-plugin-utils";
+  version = "0.0.4";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "Akuukis";
+    repo = "beancount_plugin_utils";
+    rev = "v${version}";
+    sha256 = "sha256-oyfL2K/sS4zZ7cq1P36h0dTcW1m5GUyQ9+IyZGfpb2E=";
+  };
+
+  build-system = with python3.pkgs; [
+    setuptools
+  ];
+
+  dependencies = with python3.pkgs; [
+    beancount
+  ];
+
```

Comment:
Could you enable tests with `pythonImportsCheck` and `pytestCheckHook`?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#testing-python-packages-testing-python-packages